graphql2/schema.graphql

```diff
@@ -515,6 +515,8 @@ type Mutation {
 
   createUser(input: CreateUserInput!): User
 
+  updateUserPassword(input: UpdateUserPassword!): Boolean!
```

Comment:
Can we merge this with the existing `updateUser` mutation?

Reply:
The updateUser mutation applies to the users db entry whereas the basic auth info we want to update is stored in auth_basic_users so I created a new mutation for it

Reply:
I see where you're coming from, but I think that would be leaking implementation details. In other words, the API design should be based on what is most predictable for the API layer, and not influenced by how things work under the covers.

There are always things to consider, but in this case, I feel like updating a user can include updating the user's password. From the UI side, it makes sense because it's just a single mutation backing the edit form, and from the UX perspective they are submitting the new password along with the admin true/false info.

Reply:
Ah that makes sense, will try it this way